guide/travisci.rst

```diff
@@ -48,3 +48,37 @@ OCLint is pre-installed on OS X images, however, not on linux images, so we prov
 
 An example can be found at this `github repository <https://github.com/ryuichis/oclint-cpp-travis-ci-examples>`_.
 
+C++/qmake
+---------
+
+Travis, qmake and OCLint go together fine.
+
+Here is a complete `travis.yml` of a minimal project, consisting out of a single Qt Project (`.pro`) file and a single `main.cpp` file:
+
+.. code-block:: yaml
+
+    language: cpp
+    compiler: gcc
+    sudo: required
+
+    install:
+      # install prerequisites of OCLint
+      - sudo add-apt-repository ppa:ubuntu-toolchain-r/test --yes
+      - sudo apt-get update -qq 
+      - sudo apt-get install -qq libstdc++6-4.7-dev  
+      # install OCLint
+      - wget https://github.com/oclint/oclint/releases/download/v0.10.3/oclint-0.10.3-x86_64-linux-3.13.0-74-generic.tar.gz
+      - tar -zxf oclint-0.10.3-x86_64-linux-3.13.0-74-generic.tar.gz
```

Comment:
Here is another thought, would like to run by you. 

I have a [script](https://raw.githubusercontent.com/ryuichis/oclint-cpp-travis-ci-examples/master/oclint-ci-install.sh) that I maintain as an example for running oclint on c++ projects. And I always keep that one up to date whenever there are oclint udpates. Let me know if you like the idea of running that script instead, something like [this](https://github.com/ryuichis/countly-cpp/blob/master/.travis.yml#L12).